### ISSUE TITLE
Project name not exact matched

### ISSUE THREAD

**Issue by bensheldon** [alex-page/github-project-automation-plus#24]
I really love this Github Action, but ran into an unexpected situation:

I have the following action defined:

```yml
  add_pull_request:
    name: "Add Pull Request"
    if: github.event_name == 'pull_request'
    runs-on: ubuntu-latest
    steps:
      - uses: alex-page/github-project-automation-plus@9a05133
        with:
          project: Backlog
          column: In progress
          repo-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
```

And discovered that the Action is adding the newly created PR _correctly_ to the Repo's Project named "Backlog" and column named "In progress".

...but is _unexpectedly_ also adding the PR to an Owner project named "Global Backlog" and column named "In progress".

Reading the code, it seems like the Project Name is only fuzzily matched when searching for matching Columns through the Github API. The Project Name is not subsequently use it as a filter on the result for an exact match. Columns _are_ filtered with an exact match.

Is the fuzzy-matching expected behavior? 

I think this could also be addressed by #18 to avoid collisions (either fuzzily or exact match)

**Comment by alex-page** [comment]
Ahhh good catch. This should not be fuzzy matching. 

I still want to keep the ability to add projects by name to keep this action usable for people who don't know how to get the project ID but it should only be getting exact matches.

If you want to make a pull request that would be awesome, otherwise I will fix this up in the next week or two.

**Comment by bensheldon** [comment]
Thanks for that feedback. I'll make a PR.

**Comment by alex-page** [comment]
This is released in v0.1.3. Thanks @bensheldon 💯

**Comment by bensheldon** [comment]
@alex-page woot! Thank YOU.
